x-pack/plugins/apm/public/components/app/mobile/transaction_overview/index.tsx

```diff
@@ -88,15 +89,14 @@ export function MobileTransactionOverview() {
       />
       <EuiSpacer size="s" />
       <EuiPanel hasBorder={true}>
-        <TransactionsTable
-          hideViewTransactionsLink
-          numberOfTransactionsPerPage={25}
-          showMaxTransactionGroupsExceededWarning
+        <TransactionOverviewTabs
           environment={environment}
-          kuery={kueryWithMobileFilters}
+          kueryWithMobileFilters={kueryWithMobileFilters}
           start={start}
           end={end}
-          saveTableOptionsToUrl
```

Comment:
Is this something we would like to remove? For example, If I have a tab selected in this table and I share the URL would the person opening the link landing the same state I was intending to share?